db/schema.rb

```diff
@@ -532,6 +1359,44 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "socialaccount_socialaccount", id: :serial, force: :cascade do |t|
+    t.string "provider", limit: 30, null: false
+    t.string "uid", limit: 191, null: false
+    t.timestamptz "last_login", null: false
+    t.timestamptz "date_joined", null: false
+    t.text "extra_data", null: false
+    t.integer "user_id", null: false
+    t.index ["provider", "uid"], name: "socialaccount_socialaccount_provider_uid_fc810c6e_uniq", unique: true
+    t.index ["user_id"], name: "socialaccount_socialaccount_user_id_8146e70c"
+  end
+
+  create_table "socialaccount_socialapp", id: :serial, force: :cascade do |t|
+    t.string "provider", limit: 30, null: false
+    t.string "name", limit: 40, null: false
+    t.string "client_id", limit: 191, null: false
+    t.string "secret", limit: 191, null: false
+    t.string "key", limit: 191, null: false
+  end
+
+  create_table "socialaccount_socialapp_sites", force: :cascade do |t|
+    t.integer "socialapp_id", null: false
+    t.integer "site_id", null: false
+    t.index ["site_id"], name: "socialaccount_socialapp_sites_site_id_2579dee5"
+    t.index ["socialapp_id", "site_id"], name: "socialaccount_socialapp__socialapp_id_site_id_71a9a768_uniq", unique: true
+    t.index ["socialapp_id"], name: "socialaccount_socialapp_sites_socialapp_id_97fb6e7d"
+  end
+
+  create_table "socialaccount_socialtoken", id: :serial, force: :cascade do |t|
+    t.text "token", null: false
+    t.text "token_secret", null: false
+    t.timestamptz "expires_at"
+    t.integer "account_id", null: false
+    t.integer "app_id", null: false
+    t.index ["account_id"], name: "socialaccount_socialtoken_account_id_951f210e"
+    t.index ["app_id", "account_id"], name: "socialaccount_socialtoken_app_id_account_id_fca4e0ac_uniq", unique: true
+    t.index ["app_id"], name: "socialaccount_socialtoken_app_id_636a42d7"
+  end
+
```

Comment:
This is a big schema change that is not related to this PR. Let's drop this so that our schema file stays clean 🧹